libmscore/groups.cpp

```diff
@@ -25,40 +25,46 @@ namespace Ms {
 
 static std::vector<NoteGroup> noteGroups {
       { Fraction(2,2),
-            Groups( { { 4, 512}, { 8, 272}, {12, 512}, {16, 273}, {20, 512}, {24, 272}, {28, 512} })
+            Groups( { { 4, 0x200}, { 8, 0x110}, { 12, 0x200}, { 16, 0x111}, { 20, 0x200}, { 24, 0x110}, { 28, 0x200} })
             },
-      { Fraction(4,4),
-            Groups( { { 4, 0x200}, { 8, 0x110}, {12, 0x200}, {16, 0x111}, {20, 0x200}, {24, 0x110}, {28, 0x200} })
+      { Fraction(3,2),
+            Groups( { { 4, 0x200}, { 8, 0x110}, { 12, 0x200}, { 16, 0x111}, { 20, 0x200}, { 24, 0x110}, { 28, 0x200}, { 32, 0x111}, { 36, 0x200}, { 40, 0x110}, { 44, 0x200} })
```

Comment:
It is better to keep number of spaces equal before the first number and after the last one, so if adding a space before the first one it is probably better to add one at the end of a group as well (or to remove both leading and trailing space). It looks like the code wasn't consistent about that previously but if changing this anyway it would be better to make it in a cleaner way.

Reply:
Changing it the way @emfomy did it was less of a change.
Main point though is to sort of align this better, so differences are easier to spot